social_marketing/overview/manage_social.rst

```diff
@@ -0,0 +1,79 @@
+============================================
+Manage Social Media Pages and Create Content
+============================================
+
+Once you are present on social media platforms, it is important to keep them up-to-date. However,
+that might feel like a full time job. Having a solution that allows you to create, schedule, keep
+track, engage with your audience and measure results saves you time and helps you to successfully
+execute your online strategy, from posts to results.
+
+
+Add accounts and create your feed
+=================================
+
+Click on *Add Stream* and grant the required permissions to add your accounts.
+
+.. image:: media/add_streams.png
+   :align: center
+   :alt: Click on add streams to add accounts in Odoo Social Marketing
+
+.. note::
+   You can link one Facebook page where you are the admin.
```

Comment:
"where" is for a location.

```suggestion
   You can link a Facebook page that you administrate.
```
or
```suggestion
   You can link a Facebook page for which you have the admin rights.
```

Reply:
it is "a" page
--> you can add more than one page in total. 
With "you can add one page", it implies that Odoo Social Marketing can only have one Facebook page at a time.

Also, "that" is still not correct in the way you used it in your edit.

Best of both worlds, in the way you wrote it, would be:
`   You can link a Facebook page for which you are the admin.
`